ops/terraform/services/eft/main.tf

```diff
@@ -122,6 +146,8 @@ resource "aws_s3_bucket_server_side_encryption_configuration" "this" {
       kms_master_key_id = local.kms_key_id
       sse_algorithm     = "aws:kms"
     }
+
+    bucket_key_enabled = true
```

Comment:
Oof. This could have been an expensive mistake. Thanks for addressing this.

Reply:
Yeah, I thought the above configuration achieved the same affect until I noticed in the console that the bucket key _wasn't_ enabled. Guess I interpreted `apply_server_side_encryption_by_default` as "enable this key as the default key _and_ as the bucket key" 😅.